cogboard-webapp/src/components/NavBar.js

```diff
@@ -20,7 +20,7 @@ const StyledBoardSwitcher = styled(BoardSwitcher)`
 `;
 
 const StyledAppBar = styled(AppBar)`
-  background-color: transparent;
+  background-color: #211f39;
```

Comment:
It is ok, but when we modify some components now, we split them into files in folders. Check Widget component for reference.

Reply:
Cool, thanks for recomendations. I'll stick to them next time.